config/default.py

```diff
@@ -87,6 +87,36 @@
 NETCDF_CALENDAR_TYPE = "gregorian"
```

Comment:
This file has a STREAM_ENGINE_VERSION variable that probably ought to be updated to "1.3.11".

Reply:
will update

Reply:
having a developer remember to update both the release notes file and this file may get a bit confusing over time.  The version should be able to be parsed from the release notes file, I'll write a ticket for this work.

Reply:
Done, updated version and created ticket: https://redmine.oceanobservatories.org/issues/13063